### PR TITLE
[gpstracker] Update the README.md

### PR DESCRIPTION
**Pull request by semperor**
Update the README.md on the basis of the new GPSLogger version and to be more noob-friendly.

### CODE REVIEW

File: addons/binding/org.openhab.binding.gpstracker/README.md [hunk 1]
```diff
@@ -1,19 +1,16 @@
...
 
 * [OwnTracks](https://owntracks.org/booklet/) - iOS, Android
 * [GPSLogger](https://gpslogger.app/) - Android
 
-GPS location reports are sent to openHAB using HTTP protocol.
-Please be aware that this communication uses the public network so make sure your openHAB installation is [secured](https://www.openhab.org/docs/installation/security.html#encrypted-communication) (but accessible from public internet through myopenhab.org or using a reverse proxy)
-and you configured HTTP**S** access in tracking applications.
+GPS location reports are sent to openHAB using HTTP. Please be aware that this communication uses the public network so make sure your openHAB installation is [secured](https://www.openhab.org/docs/installation/security.html#encrypted-communication) (but accessible from public internet through myopenhab.org or using a reverse proxy) and you configured HTTP**S** access in tracking applications. The easiest way to achieve this is to use the [openHAB Cloud Connector](https://www.openhab.org/addons/integrations/openhabcloud/) in conjunction with [myopenHAB.org](http://www.myopenhab.org/).
```
Comment (wborn) [inline addons/binding/org.openhab.binding.gpstracker/README.md]: Can you start each sentence on a new line in this Markdown file? That way it is easier to see changes and add review comments.

File: addons/binding/org.openhab.binding.gpstracker/README.md [hunk 2]
```diff
@@ -100,8 +79,8 @@ These dynamic channels require the following parameters:
...
 | Region Name   | String   | Region name. If the region is configured in the tracker app as well use the same name. Distance channels can also be defined as binding only regions (not configured in trackers) |
 | Region center | Location | Region center location                                                                                                                                                            |
-| Region Radius | Integer  | Geofence radius                                                                                                                                                                   |
-| Accuracy Threshold | Integer  | Location accuracy threshold (0 to disable)                                                                                                                                                                 |
+| Region Radius | Integer  | Geofence radius in meter                                                                                                                                                                 |
```
Comment (gbicskei) [inline addons/binding/org.openhab.binding.gpstracker/README.md]: This is not necessarily in meters. The binding uses Number:Length so it depends on system UOM.

File: addons/binding/org.openhab.binding.gpstracker/README.md [hunk 3]
```diff
@@ -100,8 +79,8 @@ These dynamic channels require the following parameters:
...
+| Accuracy Threshold | Integer  | Location accuracy threshold in meters (0 to disable)                                                                                                                                                                 |
```
Comment (gbicskei) [inline addons/binding/org.openhab.binding.gpstracker/README.md]: Pls see my previous comment...

File: addons/binding/org.openhab.binding.gpstracker/README.md [hunk 4]
```diff
@@ -24,52 +21,34 @@ Install [OwnTracks for Android](https://play.google.com/store/apps/details?id=or
...
 
 ### GPSLogger
 
 Install [GPSLogger for Android](https://play.google.com/store/apps/details?id=com.mendhak.gpslogger) on your device.
 After the launch, go to General Options.
-Enable Start on boot-up and Start on app launch.
+Enable **Start on boot-up** and **Start on app launch**.
 
-![Image](doc/gpslogger_1.png)
+Go to *Logging details* and enable **Log to custom URL**. If you only want to use GPSLogger for this binding, you can disable all other "*Log to*" entries. Right after enabling, the app takes you to the *Log to custom URL* settings:
```
Comment (gbicskei) [inline addons/binding/org.openhab.binding.gpstracker/README.md]: If image references are removed pls remove the image resources as well.

Comment (semperor) [reply addons/binding/org.openhab.binding.gpstracker/README.md]: done

**Comment by semperor** [comment]
Signed-off-by: Hagen Marianek semperor@web.de

**Pull request merged**